### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by vanadium23**
Hello, guys.

I view and #149 issue about tests for tinymce and want to help you. I've started on basic test infrastructure. What was done:

- Unit test for template tag
- Unit test for get_language_config
- Unit test for HTMLField
- Make changes due to pep8
- Add travis ci as building system
- Refactor old-style % formatting to .format()

What need to be done:
- Test for AdminWidget
- Test for views

I have a question about views testing. How do I suppose to test textarea_js_view or preview? In docstring it is described, that they are just rendering templates, but there are no such templates...

### CODE REVIEW

**Comment by Natim** [comment]
@vanadium23 this is amazing, I will go through your PR today. Thanks a lot for starting this!

**Comment by Natim** [comment]
> How do I suppose to test textarea_js_view or preview?

We can test that all the mandatory javascript files are loaded in the generated html content.

Here are the templates files we have: https://github.com/aljosa/django-tinymce/tree/master/tinymce/templates/tinymce

You are right tinymce_textareas doesn't seems to be used anywhere.
I propose that we simply remove that code that seems to be dead.

**Comment by Natim** [comment]
I propose to display the coverage report.

**Comment by vanadium23** [comment]
Add coverage report via coveralls.
Add tests for views and widget.

Compressor code seems to messy. I'll do it later in separate request.

**Comment by vanadium23** [comment]
Merge?

File: tinymce/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            title='Test Title',
+        )
+        response = self.client.get('/tinymce/flatpages_link_list/')
+        result_ok = b'var tinyMCELinkList = [["Test Title", "/test/url"]];'
+        self.assertEqual(200, response.status_code)
+        self.assertEqual('application/x-javascript', response['Content-Type'])
+        self.assertEqual(result_ok, response.content)
+
+    def test_compressor(self):
+        # TODO: implement test
```
Comment (Natim) [inline tinymce/tests/test_views.py]: Can you add a skip decorator to tests that aren't implemented yet?

**Comment by Natim** [comment]
Yes I am definitely going to merge this. Please add yourself to the contributor file as well.

Really thanks a lot this is really great!

**Comment by vanadium23** [comment]
Add skip and contributors. Thank you for giving me a chance. (:

**Comment by Natim** [comment]
> Thank you for giving me a chance.

You actually did a really great job here. Thank you for bringing tests to django-tinymce.

**Pull request merged**